crates/wash-cli/src/par.rs

```diff
@@ -13,6 +13,11 @@ use wash_lib::cli::{extract_keypair, inspect, par, CommandOutput, OutputKind};
 
 const GZIP_MAGIC: [u8; 2] = [0x1f, 0x8b];
 
+/// Helper function for detecting the arch used by the current machine
+fn detect_arch() -> String {
+    format!("{}-{}", std::env::consts::ARCH, std::env::consts::OS)
+}
```

Comment:
Just to be doubly sure, does this ARCH/OS convention match our standard set?
```
  x86_64-macos
  aarch64-macos
  x86_64-windows
  aarch64-linux
  x86_64-linux
```
I know in some cases it can report `arm64` here instead of `aarch64` and `darwin` instead of `macos`

Reply:
It does! I tested on my Linux laptop/desktop (arch btw) and my Macbook

Reply:
For reference:

https://doc.rust-lang.org/1.72.1/std/env/consts/constant.ARCH.html
https://doc.rust-lang.org/1.72.1/std/env/consts/constant.OS.html